scripts/install-binary.mjs

```diff
@@ -2,6 +2,12 @@
 
 'use strict'
 
+// First check if the user has configured a local jq binary
+if (process.env.npm_config_jq_path) {
+  console.log(`Using configured local jq binary: ${process.env.npm_config_jq_path}`)
+  process.exit(0)
+}
```

Comment:
This isn't what we want. Exiting in case of not setting the variable breaks all other workflows